js/ui/appDisplay.js

```diff
@@ -1989,11 +1989,13 @@ var RenameFolderMenu = class RenameFolderMenu extends PopupMenu.PopupMenu {
         this.addMenuItem(separator);
 
         // Add the "Remove from desktop" menu item at the end.
-        let item = new PopupMenu.PopupMenuItem(_("Remove from desktop"));
-        this.addMenuItem(item);
-        item.connect('activate', () => {
-            this._iconGridLayout.removeIcon(source.id, true);
-        });
+        // FIXME: re-enable menu below once the undo action for removing
```

Comment:
Do we actually intend to do this? I suggest that we should just remove the code. If we're leaving it in place we should have a ticket for this pending work, and refer to this here.

Reply:
Fair enough, updated to remove the code.